workspaces/explore/.changeset/honest-bottles-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-explore-backend': minor
+---
+
+Remove support for old backend system
```

Comment:
We only marked this as deprecated on December 3rd, it's only been two weeks. Can we hold off on this until the new year?

Reply:
Sure, I'm mostly interested in getting rid of `backend-common`